### DISCUSSION TITLE
Where do manifests of new alpha/beta/prerelease software belong?

### DISCUSSION THREAD

**Discussion by LucasWagler** [ScoopInstaller/Extras#13889] [Q&A] [answered]
When no stable release yet exists, what bucket should house alpha/beta/prerelease software manifests?

- Extras bucket prefers stable versions, very understandably
- Versions bucket requires a stable counterpart

Early versions of exciting new software seem to slip through the cracks of the official buckets. Is there a home for such manifests? If not, can we create one?

Example: #13888

**Comment by niheaven** [comment, marked as answer]
If a stable version is unavailable, placing it in the `extras` bucket is acceptable.

**Reply by SegoCode** [reply]
Yes I agree, but it would have to be flagged in some way, and have a resilient github Action for updates, since the installers of these versions have breaking changes every new releases usually
